README.md

```diff
@@ -157,6 +175,20 @@ The optional configuration JSON file defaults to the `convertToStdVCF.json` resi
 
   * patch: Remove all code related to PyPy and hts-python (including `copysam.py` and `PYPY_OR_PYTHON_BINARY`)
 
+
+* 3.0.0
+
+  * Major
+    * Support for hg38/GRCh38 reference genome and variant calling from ALT and HLA contigs.
```

Comment:
Are there any new mandatory configuration values? At least list them. Details should be in the XML.
Was the semantics of conf. values changed?